src/module.c

```diff
@@ -1488,35 +1488,95 @@ int RM_ReplyWithSimpleString(RedisModuleCtx *ctx, const char *msg) {
     return REDISMODULE_OK;
 }
 
+#define COLLECTION_REPLY_ARRAY      1
+#define COLLECTION_REPLY_MAP        2
+#define COLLECTION_REPLY_SET        3
+#define COLLECTION_REPLY_ATTRIBUTE  4
+#define COLLECTION_REPLY_PUSH       5
```

Comment:
BigNumber?

Reply:
yeah, let's add it instead of push for now.

Reply:
Bi`gNumber` is a collection?

Reply:
This requires the merge of PR #9202 which adds `addReplyBigNum`.

Reply:
you're right. this "enum" is just about collections.. big number has nothing to do with this.